src/app/introduction/activity-3/activity-3.component.ts

```diff
@@ -19,7 +20,10 @@ export class PicturePuzzleComponent implements OnInit {
     // "#video" is the name of the template variable in the video element
     @ViewChild('video') video: any;
 
-    private PUZZLE_DIFFICULTY = 4;
+    private puzzleDifficulty: number;
+    private readonly DIFFICULTY_LIST = [4, 6, 8];
+    private levels = ['level1', 'level2', 'level3'];
```

Comment:
can be a constant?

Reply:
yep that and naming convention

Reply:
Yes. This would change when the previous PR is merged.